main/2021/CG-08-03.md

```diff
@@ -27,8 +27,16 @@ Installation is required, see the calendar invite.
     1. Review of action items from prior meeting.
     1. Update on [Branch Hinting](https://github.com/WebAssembly/branch-hinting) [20 min].
         1. Poll for phase 3
-    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [30 min].
-        1. Poll for maintaining single list-of-USV `string` type
+    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [40 min].
+        1. Poll for supporting [UTF-16](https://github.com/WebAssembly/interface-types/issues/136#issuecomment-861799460) lifting and lowering
+        1. Poll for supporting compact UTF-16 (Latin1) lifting and lowering
+        1. Potential formal objection: A "**single** list-of-USV `string` type" may conflict with our [high-level goals](https://webassembly.org/docs/high-level-goals/), with [what](https://webassembly.org/) we [communicate](https://www.w3.org/2020/03/webassembly-wg-charter), and [envisioned](https://www.youtube.com/watch?v=ZKydJPRosa0). In particular:
```

Comment:
Should this be on the agenda? If you want to formally object, then it could be done as part of the poll below.

Reply:
To be honest, I don't know why we are polling in the first place with my concerns and suggestions largely unaddressed.

Reply:
Although unanimity is the goal, it's by no means required of the [consensus process](https://github.com/WebAssembly/meetings/blob/main/process/consensus.md).  There are a number of times in the past when there was a non-unanimous poll, but, to make progress, the CG chairs determined overall consensus was reached and moved on.  In any case, agreed with Piotr that objections are things to raise in the meeting, not in the agenda item which, according to the [process](https://github.com/WebAssembly/meetings/blob/main/process/consensus.md#in-person-meeting-consensus), just states poll text as determined by the proposal champion(s).

Reply:
That's overall pretty clever, yeah.